.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: node_js
+dist: bionic
 
-node_js: 12
+node_js: 16
 
 # enable c++11/14 builds
 addons:
   apt:
     sources: [ 'ubuntu-toolchain-r-test' ]
-    packages: [ 'libstdc++-5-dev' ]
+    packages: [ 'libstdc++-5-dev', 'python3.6' ]
```

Comment:
Probably not needed to have 'python3.6' separately, since you use bionic. Check this.

Reply:
Done.